### PR TITLE
Add an option to send keyword arguments when creating a Namespace

### PR DESCRIPTION
**Pull request by Anatw**

### CODE REVIEW

**Comment by myakove** [comment]
Please update the title, the timeout wasn't changed. It was not exists. (you add it)

**Comment by myakove** [comment]
> In several network test modules, the 240 seconds timeout was not enough.
> 
> ##### Short description:
> 
> ##### More details:
> 
> ##### What this PR does / why we need it:
> 
> ##### Which issue(s) this PR fixes:
> 
> ##### Special notes for reviewer:
> 
> ##### Bug:
> 

Please remove the reference for 'tests'.

File: ocp_resources/namespace.py [hunk 1]
```diff
@@ -25,9 +25,14 @@ def __init__(
...
         yaml_file=None,
+        timeout=240,
     ):
         super().__init__(
-            name=name, client=client, teardown=teardown, yaml_file=yaml_file
+            name=name,
+            client=client,
+            teardown=teardown,
+            yaml_file=yaml_file,
+            timeout=timeout,
```
Comment (myakove) [inline ocp_resources/namespace.py:35]: Please pass kwargs instead of timeout

File: ocp_resources/namespace.py [hunk 2]
```diff
@@ -25,9 +25,14 @@ def __init__(
         teardown=True,
         label=None,
         yaml_file=None,
+        timeout=240,
```
Comment (myakove) [inline ocp_resources/namespace.py:28]: What is the default timeout in Resource?

**Review by myakove — CHANGES_REQUESTED** [review]

**Review by rnetser — CHANGES_REQUESTED** [review]
This repo is not tied to openshift virtualization; please update the description

File: ocp_resources/namespace.py (hunk 2 above)
Comment (Anatw) [reply ocp_resources/namespace.py:28]: 240 seconds. I'll import it from Resource to make it bond to it.

File: ocp_resources/namespace.py (hunk 1 above)
Comment (Anatw) [reply ocp_resources/namespace.py:35]: ack.

File: ocp_resources/namespace.py [hunk 3]
```diff
@@ -1,6 +1,5 @@
 from ocp_resources.logger import get_logger
-from ocp_resources.resource import Resource
-
+from ocp_resources.resource import Resource, TIMEOUT
```
Comment (dbasunag) [inline ocp_resources/namespace.py:2]: TIMEOUT is not needed.

**Review by dbasunag — CHANGES_REQUESTED** [review]

**Review by rnetser — CHANGES_REQUESTED** [review]
Please update the PR's title and description; it is not related to timeout

File: ocp_resources/namespace.py (hunk 3 above)
Comment (Anatw) [reply ocp_resources/namespace.py:2]: I want to use it instead of '240' so that the two will be bonded.

Comment (Anatw) [reply ocp_resources/namespace.py:2]: ack.

**Comment by Anatw** [comment]
!verified

**Comment by rnetser** [comment]
@Anatw please see https://github.com/RedHatQE/openshift-python-wrapper/pull/823#pullrequestreview-1113746624

**Pull request closed without merging**